### PR TITLE
Start new unreleased section in changelog

### PR DESCRIPTION
**Pull request by diego-plan9**
## Description

Small post-release tweak in order to start a new section in the changelog after the 0.4.0 release, moving the entries since then to this new section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ The format is based on [Keep a Changelog], and this project adheres to
 * `Fixed` for any bug fixes.
 * `Security` in case of vulnerabilities.
 
+## [UNRELEASED]
+
+### Added
+
+* Load model state dict into a new model with modified `RPUConfig`. (\#276)
+* Visualization for noise models for analog inference hardware simulation. (#278)
```
Comment (maljoras) [inline CHANGELOG.md:20]: could you change `#` --> `\#`  to make it consistent?

**Review by maljoras — APPROVED** [review]
Many thanks @diego-plan9!

File: CHANGELOG.md (hunk 1 above)
Comment (diego-plan9) [reply CHANGELOG.md]: Absolutely! Fixed at 7435466

**Review by maljoras — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (maljoras) [reply CHANGELOG.md]: Many thanks!

**Pull request merged**